src/Type/Row/Homogeneous.purs

```diff
@@ -0,0 +1,22 @@
+module Type.Row.Homogeneous
+  ( class FieldOf
+  , class Homogeneous
+  ) where
+
+-- | Ensure that every field in a row has the same type.
```

Comment:
This comment should be removed, I think.

Reply:
Yes. Strange things like this sometimes happen as a result of imports being managed by the PureScript Emacs layer.